model/ner_model/project.yml

```diff
@@ -32,9 +32,9 @@ assets:
 workflows:
   all:
     - convert
-    - create-config
     - train
-    - evaluate
+    - evaluate1
```

Comment:
I see everywhere there is now an evaluate1 and evaluate 2, also with some model specific counting. Doing this in the project will ensure component 1 and component 2 are run at the same time, but only for evaluation. The handling of the seperate models is done during training in the training notebook/file with all paths accordingly. Doing this in the project is not working correctly. It is possible to do this in the separate file for metrics => visuaize_results.ipynb